tests/functional/UseCase/Table/PreviewTableTest.php

```diff
@@ -769,8 +820,7 @@ private function dropTable(string $databaseName, string $tableName): void
      *     changedUntil?: string,
      *     fulltextSearch?: string,
      *     whereFilters?: TableWhereFilter[],
-     *     orderBy?: array<string, int>,
-     *     orderByDataType?: int,
+     *     orderBy?: PreviewTableOrderBy[],
```

Comment:
to by nemelo byt nullable ne?

Reply:
to neni nullalble, ale ze tam ten klic muze chybet. To je vlastni pole z providera, ktery se v ty metode sklada do objektu...

Reply:
souvisi s tim `isset($commandInput['orderBy']))` nize:
```php
        if (isset($commandInput['orderBy'])) {
            $orderBy = new RepeatedField(GPBType::MESSAGE, PreviewTableOrderBy::class);
            foreach ($commandInput['orderBy'] as $orderByOrig) {
                $orderBy[] = $orderByOrig;
            }
            $command->setOrderBy($orderBy);
        }
```